src/neo-vm/Script.cs

```diff
@@ -9,6 +9,28 @@ public class Script
         private readonly byte[] _value;
         private readonly ICrypto _crypto;
 
+        /// <summary>
```

Comment:
I prefer the constructor after the properties

Reply:
These C# guys...... xD